app/models/peep.rb

```diff
@@ -0,0 +1,14 @@
+require 'data_mapper'
+require 'dm-postgres-adapter'
+
+class Peep
+  include DataMapper::Resource
+
+  has n, :comments, through: Resource
```

Comment:
what type of relationship does this define?  Is it correct?

Isn't there another relationship that `Peep` should have?